### PR TITLE
Namespace pagination

### PR DESCRIPTION
**Pull request by laoneo**
Namespace pagination.

### CODE REVIEW

File: libraries/src/Joomla/CMS/Pagination/Pagination.php [hunk 1]
```diff
@@ -346,11 +349,11 @@ public function getPagesLinks()
 
 			/*
 			 * @deprecated The list rendering is now a layout.
```
Comment (brianteeman) [inline libraries/src/Joomla/CMS/Pagination/Pagination.php]: wouldnt this be a good time to add a version to this deprecated notcice

**Comment by mbabker** [comment]
The classmap conflicts since I merged the menu PR.

File: libraries/src/Joomla/CMS/Pagination/Pagination.php (hunk 1 above)
Comment (laoneo) [inline libraries/src/Joomla/CMS/Pagination/Pagination.php]: Which version?

Comment (brianteeman) [inline libraries/src/Joomla/CMS/Pagination/Pagination.php]: pass

**Comment by laoneo** [comment]
Conflicts solved

**Pull request merged**